dsl/config/AppConfig.dhall

```diff
@@ -0,0 +1,39 @@
+{-
+    Sample unipipe service broker configuration
+    using ssh-key method to access the git repo
+    
+    Application configuration
+    basic-auth-username :
+        The service broker API itself is secured via HTTP Basic Auth.
+        The basic auth username for requests against the API.
+    basic-auth-password :
+        The basic auth password for requests against the API.
+
+    Git access configuration
+         
+         SSH configuration :
+            local : 
+                The path where the local Git Repo shall be created/used. Defaults to tmp/git
+            remote : 
+                The remote Git repository to push the repo to
+            ssh-key : 
+                This is the SSH key to be used for accessing the remote repo. Linebreaks must be replaced with spaces
```

Comment:
s: Please be as concise as possible of what should go here. What is the expected format of the key? How would it look like?